gn prior to ad1e5ce10f06ef9a1a1d91b2e48231d6b3eecbe2 (20230426) doesn't support the
mnemonic key in actions and as of this writing 3.18-stable is on
5a004f9427a050c6c393c07ddb85cba8ff3849fa (20230419). This is a harmless fix that
just makes the build output a bit noisier.
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -1694,8 +1694,6 @@
   python_path_root = "${root_out_dir}/pyproto"
   python_path_proto = "${python_path_root}/third_party/blink/renderer/core/lcp_critical_path_predictor"
 
-  mnemonic = "ELOC_PROTO"
-
   source_dir = "lcp_critical_path_predictor/test_proto"
   sources = rebase_path([ "lcp_image_id.asciipb" ], "", source_dir)
 
